tests/AutoReview/ProjectCodeTest.php

```diff
@@ -361,6 +394,25 @@ public function provideDataProviderMethodNameCases()
         return $data;
     }
 
+    public function provideClassesWherePregFunctionsAreForbiddenCases()
+    {
+        if (extension_loaded('xdebug') && false === getenv('CI')) {
+            $this->markTestSkipped('Data provider too slow when Xdebug is loaded.');
```

Comment:
Here I think the slow part is the test itself, not its data provider.

Reply:
What I meant is that skip message is not totally accurate :p

Reply:
m'kay, done